include/splinterdb/platform_linux/public_platform.h

```diff
@@ -68,7 +68,9 @@ typedef int32 bool;
 typedef uint8 bool8;
 
 // See platform.c
-extern FILE *Platform_stdout_fh; // File handle for stdout
-extern FILE *Platform_stderr_fh; // File handle for stderr
+typedef FILE platform_log_handle;
+extern platform_log_handle
+   *Platform_default_log_handle;                       // File handle for stdout
```

Comment:
Minor request: For readability, if you can comment these two lines as below, it will be better. And clang-format will not complain / re-indent then:

```
 71 typedef FILE                platform_log_handle;
 72 extern platform_log_handle *Platform_default_log_handle; // stdout FILE handle
 73 extern platform_log_handle *Platform_error_log_handle;   // stderr FILE handle